src/components/ComboBox/ComboBox.tsx

```diff
@@ -36,13 +35,46 @@ export interface ComboBoxProps {
    */
   placeholder?: string;
   /**
-   * Input value for the ComboBox.
+   * Name attribute for form submission
    */
-  inputValue: string;
+  name?: string;
   /**
-   * Passed in function to handle input value changes.
+   * Value for the selected option
    */
-  setInputValue: (value: string) => void;
+  value?: string;
+  /**
+   * Handler for value changes
+   */
+  onChange?: (value: string, option: ComboBoxOption | null) => void;
+  /**
+   * Input value (display text)
+   * Only needed for controlled usage, otherwise managed internally
+   */
+  inputValue?: string;
```

Comment:
I don't understand why we need a `inputValue` and `setInputValue` in addition to the `onChange` and `value` prop?